src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random randomColor;
+
+    public ColorSupplier() {
+        this.randomColor = new Random();
+    }
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[randomColor.nextInt(Color.values().length)].toString();
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)
Use name() for getting String representation of enum constants

And split logic into 2 lines - 5 method calls is too much.

Reply:
Agree, thx